libs/ledger-live-common/src/families/evm/editTransaction/isStrategyDisabled.ts

```diff
@@ -0,0 +1,37 @@
+import { getMinFees } from "@ledgerhq/coin-evm/getMinEditTransactionFees";
+import type { FeeData, Transaction } from "@ledgerhq/coin-evm/types/index";
+
+/**
+ * A strategy is disabled if it's fees are lower than the minimum fees
+ */
+export const isStrategyDisabled = ({
+  transaction,
+  feeData,
+}: {
+  transaction: Transaction;
+  feeData: FeeData;
+}): boolean => {
+  const minFees = getMinFees({ transaction });
+
+  if (transaction.type === 2) {
+    if (
+      !feeData.maxFeePerGas ||
+      !feeData.maxPriorityFeePerGas ||
+      !minFees.maxFeePerGas ||
+      !minFees.maxPriorityFeePerGas
+    ) {
+      return false;
```

Comment:
it seems that we will never arrive there. maybe we can only just use invariant to assert feeData.maxFeePerGas, feeData.maxPriorityFeePerGas, minFees.maxFeePerGas and minFees.maxPriorityFeePerGas are not undefined

Reply:
We can arrive here in multiple scenarios, that are not necessarily errors (for example, maybe the server did not return a gasOption for the selected strategy, or maybe the gasOptions are not yet fully loaded). And if we use invariant it will throw an error, which we don't really want here.